### PR TITLE
[ERSSUP-55974]-[]-[Convert to unversioned URLs]-[DMW]

### PR DESCRIPTION
**Pull request by nhsd-david-wass**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: sandbox/README.md [hunk 1]
```diff
@@ -19,18 +19,18 @@ Redeploy the API Proxy. See the main [README.md](../README.md).
...
-- [x] GET     `/STU3/v1/Binary/{attachmentLogicalID}`
-- [x] GET     `/STU3/v1/ReferralRequest/{ubrn}`
-- [x] GET     `/STU3/v1/ReferralRequest/{ubrn}/_history/{version}`
-- [x] POST    `/STU3/v1/ReferralRequest/{ubrn}/$ers.maintainReferralLetter`
-- [x] POST    `/STU3/v1/Appointment`
-- [x] POST    `/STU3/v1/ReferralRequest/{ubrn}/$ers.generateCRI`
+- [x] POST    `/FHIR/STU3/ReferralRequest/$ers.createReferral`
+- [x] POST    `/FHIR/STU3/ReferralRequest/$ers.createReferralAndSendForTriage`
+- [x] POST    `/FHIR/STU3/HealthcareService/$ers.searchHealthcareServicesForPatient`
+- [x] GET     `/FHIR/STU3/CodeSystem/{codeSystemType}`
```
Comment (nhsd-david-wass) [inline sandbox/README.md:25]: /azp run

File: sandbox/src/mocks/createReferral/requests/MinimalRequest.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
               {
                 "name": "specialty",
                 "valueCoding": {
-                  "system": "_baseUrl_/STU3/v1/CodeSystem/SPECIALTY",
+                  "system": "_baseUrl_/STU3/CodeSystem/SPECIALTY",
```
Comment (nhsd-jack-wainwright) [inline sandbox/src/mocks/createReferral/requests/MinimalRequest.json:58]: These endpoints should also include "FHIR" I think, "_baseUrl_/FHIR/STU3/CodeSystem/SPECIALTY"

File: sandbox/README.md [hunk 3]
```diff
@@ -19,18 +19,18 @@ Redeploy the API Proxy. See the main [README.md](../README.md).
...
-- [x] GET     `/R4/v1/PractitionerRole`
-- [x] POST    `/STU3/v1/Binary`
-- [x] POST    `/STU3/v1/ReferralRequest/{ubrn}/$ers.generatePatientLetter`
-- [x] GET     `/STU3/v1/Binary/{attachmentLogicalID}`
-- [x] GET     `/STU3/v1/ReferralRequest/{ubrn}`
-- [x] GET     `/STU3/v1/ReferralRequest/{ubrn}/_history/{version}`
-- [x] POST    `/STU3/v1/ReferralRequest/{ubrn}/$ers.maintainReferralLetter`
-- [x] POST    `/STU3/v1/Appointment`
-- [x] POST    `/STU3/v1/ReferralRequest/{ubrn}/$ers.generateCRI`
+- [x] POST    `/STU3/ReferralRequest/$ers.createReferral`
```
[the remark this answers is not in the document]
Comment (nhsd-david-wass) [reply sandbox/README.md:22]: We have moved the FHIR to the end of the base URL specified in the main e-referrals-service-api.yaml file

File: sandbox/src/mocks/createReferral/requests/MinimalRequest.json (hunk 2 above)
Comment (nhsd-david-wass) [reply sandbox/src/mocks/createReferral/requests/MinimalRequest.json:58]: As above, the base URL in the main spec has had the FHIR appended

**Comment by nhsd-david-wass** [comment]
Changed to branch from develop

**Pull request closed without merging**